numba_cuda/numba/cuda/cudadrv/libs.py

```diff
@@ -2,8 +2,7 @@
 
 CUDA Toolkit libraries can be available via either:
 
-- the `cuda-nvcc` and `cuda-nvrtc` conda packages for CUDA 12,
-- the `cudatoolkit` conda package for CUDA 11,
+- the `cuda-nvcc` and `cuda-nvrtc` conda packages,
```

Comment:
I believe that with some effort, we should be able to support CUDA 13 from wheels too. CUDA 13 ships a full compiler in the `nvidia-cuda-nvcc` package. That can be out of scope for the first pass of CUDA 13 support, but will be important for RAPIDS over time so we're no longer tied to system CTK packages when using wheels.

Reply:
We have the test-wheel-deps-wheels test that tests this, so I think it is supported, but through the cuda-python bindings instead of the ctypes bindings that `libs.py` is in support of. So I'll leave this as-is for now, on the basis that it should be removed once the ctypes binding is removed.